pyradiosky/skymodel.py

```diff
@@ -654,7 +660,7 @@ def update_positions(self, time, telescope_location):
 
         if not isinstance(telescope_location, (EarthLocation, MoonLocation)):
 
-            errm = "telescope_location must be an astropy EarthLocation object"
+            errm = "telescope_location must be an :class:`astropy.EarthLocation` object"
             if hasmoon:
                 errm += " or a lunarsky MoonLocation object "
```

Comment:
This can be replaced with `:class:lunarsky.MoonLocation`.

Reply:
updated.